vllm/utils.py

```diff
@@ -2176,7 +2176,32 @@ def _check_multiproc_method():
         os.environ["VLLM_WORKER_MULTIPROC_METHOD"] = "spawn"
 
 
-def get_mp_context():
+def get_mp_context(
+    override_mp_method: Optional[str] = None,
+    override_reason: Optional[str] = None,
+):
+    """Get a multiprocessing context.
+    If override_mp_method is provided, it will be used instead of the
+    VLLM_WORKER_MULTIPROC_METHOD environment variable.
```

Comment:
can you add an example on what the allowed override_mp_methods are? e.g. spawn, fork?